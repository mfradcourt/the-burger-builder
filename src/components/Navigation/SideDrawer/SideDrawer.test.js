import React from 'react';
import { shallow } from "enzyme";

import SideDrawer from './SideDrawer';

describe('<SideDrawer />', () => {
  let wrapper;

  it('should render SideDrawer as close', () => {
    wrapper = shallow(
      <SideDrawer/>
    );

    expect(wrapper).toMatchSnapshot();
  });

  it('should render SideDrawer as open', () => {
    wrapper = shallow(
      <SideDrawer/>
    );

    wrapper.setProps({open: true})

    expect(wrapper).toMatchSnapshot();
  });
});
