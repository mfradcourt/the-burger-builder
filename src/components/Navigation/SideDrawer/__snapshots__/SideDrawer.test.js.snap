// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<SideDrawer /> should render SideDrawer as close 1`] = `
<Fragment>
  <backdrop />
  <div
    className="SideDrawer Close"
  >
    <div
      className="Logo"
    >
      <logo />
    </div>
    <nav>
      <navigationItems />
    </nav>
  </div>
</Fragment>
`;

exports[`<SideDrawer /> should render SideDrawer as open 1`] = `
<Fragment>
  <backdrop
    show={true}
  />
  <div
    className="SideDrawer Open"
  >
    <div
      className="Logo"
    >
      <logo />
    </div>
    <nav>
      <navigationItems />
    </nav>
  </div>
</Fragment>
`;
